crates/ruff_python_ast/src/visitor/transformer.rs

```diff
@@ -652,14 +653,35 @@ pub fn walk_type_param<V: Transformer + ?Sized>(visitor: &V, type_param: &mut Ty
     match type_param {
         TypeParam::TypeVar(TypeParamTypeVar {
             bound,
+            default_value,
             name: _,
             range: _,
         }) => {
             if let Some(expr) = bound {
                 visitor.visit_expr(expr);
             }
+            if let Some(expr) = default_value {
+                visitor.visit_expr(expr);
+            }
```

Comment:
Same question as for the `Visitor` trait w.r.t. the evaluation-order.

Reply:
As above